docs/src/main/paradox/release-notes/2.0.x.md

```diff
@@ -16,6 +16,57 @@ The Alpakka Kafka 2.0 series features
 * Removal of some API which was deprecated since 1.0.0 or earlier.
 * Internal filter to avoid emitting records of revoked partitions [#946](https://github.com/akka/alpakka-kafka/issues/946) and [#992](https://github.com/akka/alpakka-kafka/issues/992)
 
+# 2.0.3
+
+Released: 2020-05-14
+
+## Notable changes since 2.0.2
+
+- @ref[Akka Cluster Sharding](../cluster-sharding.md) integration to co-locate Kafka partition reading with Akka Cluster shards by [@seglo](https://github.com/seglo) and [@chbatey](https://github.com/chbatey)
+- Retry committing of offsets when failure is marked as retriable [#1111](https://github.com/akka/alpakka-kafka/issues/1111) by [@ennru](https://github.com/ennru)
+- Delay commits until the next offset is observed [#1093](https://github.com/akka/alpakka-kafka/issues/1093) by [@seglo](https://github.com/seglo)
+- Non-streaming producer wrapper [#1085](https://github.com/akka/alpakka-kafka/issues/1085) by [@ennru](https://github.com/ennru)
+- Drain partial offset commit batches on upstream failure [#1058](https://github.com/akka/alpakka-kafka/issues/1058) by [@sdudzin](https://github.com/sdudzin)
+- Kafka client 2.4.1 [#1103](https://github.com/akka/alpakka-kafka/issues/1103) by [@ennru](https://github.com/ennru)
+
+
+## Alpakka Kafka core
+
+- Committing: retry commits marked as retriable [#1111](https://github.com/akka/alpakka-kafka/issues/1111) by [@ennru](https://github.com/ennru)
+- Build: Jackson 2.10.4 and sbt plugin updates [#1112](https://github.com/akka/alpakka-kafka/issues/1112) by [@ennru](https://github.com/ennru)
+- Security: mask passwords in settings' toString [#1110](https://github.com/akka/alpakka-kafka/issues/1110) by [@ennru](https://github.com/ennru)
+- Commit when next offset is observed [#1093](https://github.com/akka/alpakka-kafka/issues/1093) by [@seglo](https://github.com/seglo)
+- Non-streaming producer wrapper [#1085](https://github.com/akka/alpakka-kafka/issues/1085) by [@ennru](https://github.com/ennru)
+- Simplify commit replying [#1095](https://github.com/akka/alpakka-kafka/issues/1095) by [@ennru](https://github.com/ennru)
+- DrainingControl: creation in toMat [#1084](https://github.com/akka/alpakka-kafka/issues/1084) by [@ennru](https://github.com/ennru)
+- Simpler use with the new actors API [#1088](https://github.com/akka/alpakka-kafka/issues/1088) by [@ennru](https://github.com/ennru)
+- Cluster-sharding: single listener per entity type key + Java DSL [#1080](https://github.com/akka/alpakka-kafka/issues/1080) by [@chbatey](https://github.com/chbatey)
```

Comment:
It would be nice to give akka-stream-kafka-cluster-sharding its own section

Reply:
It got a link to the docs in "Notable changes".